observability/akka-http-metrics/src/test/suite/scala/com/daml/metrics/akkahttp/AkkaHttpMetricsSpec.scala

```diff
@@ -0,0 +1,621 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics.akkahttp
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import com.google.common.primitives.Bytes
+
+import akka.util.ByteString
+import akka.stream.scaladsl.Source
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.Directives
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.model.{
+  HttpRequest,
+  HttpResponse,
+  HttpEntity,
+  RequestEntity,
+  ResponseEntity,
+  StatusCodes,
+  ContentTypes,
+}
+import com.daml.metrics.akkahttp.AkkaUtils._
+
+import com.daml.metrics.api.MetricsContext
+import com.daml.metrics.api.MetricName
+import com.daml.metrics.api.MetricHandle.{Counter, Timer}
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+
+class AkkaHttpMetricsSpec extends AnyWordSpec with Matchers with ScalatestRouteTest {
+
+  private val byteArray1 = Array[Byte](1, 12, -7, -124, 0, 127)
+
+  private val byteArray2 = Array[Byte](-4, -3, -2, -1, 0, 1, 2, 3, 4)
```

Comment:
Just a suggestion to make the test cases easier to read, i.e., to rename these and wrap them into a `ByteString` as that's what we need eventually:
```suggestion
  private val aByteString = ByteString(Array[Byte](1, 12, -7, -124, 0, 127))
  private val anotherByteString = ByteString(Array[Byte](-4, -3, -2, -1, 0, 1, 2, 3, 4))
```